tests/unit_and_functional/conftest.py

```diff
@@ -75,3 +77,22 @@ def k8s_env_vars() -> Iterable[bool]:
     finally:
         yield True
     del os.environ['KUBERNETES_SERVICE_HOST']
+
+def onerror(func, path, exc_info):
+    """
+    Error handler for ``shutil.rmtree``.
+
+    If the error is due to an access error (read only file)
+    it attempts to add write permission and then retries.
+
+    If the error is for another reason it re-raises the error.
+
+    Usage : ``shutil.rmtree(path, onerror=onerror)``
+    """
+    import stat
+    if not os.access(path, os.W_OK):
+        # Is the error an access error ?
+        os.chmod(path, stat.S_IWUSR)
+        func(path)
+    else:
+        raise
```

Comment:
You need to specify what type of exception to raise - e.g. `raise RuntimeError('this did not work)`. 

Without this, you'll get an exception raised due to the bad exception definition - e.g. `RuntimeError: No active exception to reraise`

Reply:
👍